keyboards/planck/keymaps/ipetepete/keymap.c

```diff
@@ -0,0 +1,243 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _GAM3R 2
+#define _SPCFN 3
+#define _LOWER 4
+#define _RAISE 5
+#define _GAMUP 8
+#define _FUNC 10
+#define _MOUS 12
+#define _ADJUST 16
+
+#define KC_SPFN LT(_SPCFN, KC_SPC) // press for space, hold for function layer (aka spacefn)
+#define KC_BKFN LT(_SPCFN, KC_BSPC)
+// use tap dancing to set caps lock
+#define TD_ESC 0
+#define HYPER RCTL(RALT(RSFT(KC_RGUI)))
+#define KC_TABM LT(_MOUS, KC_TAB)
+#define ENDLIN LGUI(KC_RGHT)
+#define SRTLIN LGUI(KC_LEFT)
+#define SPCRGT LCTL(KC_RGHT)
+#define SPCLFT LCTL(KC_LEFT)
+#define ENTSFT SFT_T(KC_ENT)
+#define CTLESC RCTL_T(KC_ESC) //CTL_T(KC_ESC)
+#define VOLUP  S(LALT(KC_VOLU))
+#define VOLDN  S(LALT(KC_VOLD))
+#define POWER S(LCTL(KC_POWER))
+
+enum planck_keycodes {
+  QWERTY = 0,
+  GAM3R,
+  GAMEUP,
+  LOWER,
+  RAISE,
+  FUNC,
+  BELOW,
+  ABOVE,
+  TABM
+};
+
+enum keyboard_marcos {
+  MACRO_QWERTY = 0,
+  MACRO_GAM3R,
+  MACRO_GAMEUP,
+  MACRO_LOWER,
+  MACRO_RAISE,
+  MACRO_FUNC,
+  MACRO_BELOW,
+  MACRO_ABOVE,
+  MACRO_TABM
+};
+
+#define M_QWERTY  M(MACRO_QWERTY)
+#define M_GAMR    M(MACRO_GAM3R)
+#define M_GAMEUP  M(MACRO_GAMEUP)
+#define M_LOWER   M(MACRO_LOWER)
+#define M_RAISE   M(MACRO_RAISE)
+#define M_FUNC    M(MACRO_FUNC)
+#define M_BELOW   M(MACRO_BELOW)
+#define M_ABOVE   M(MACRO_ABOVE)
+#define M_TABM    M(MACRO_TABM)
+
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```